### DISCUSSION TITLE
Colour formatting is different to Patricks

### DISCUSSION THREAD

**Discussion by rorymcguire42** [Cyfrin/foundry-full-course-cu#937] [General]
Hi - this may be trivial but it's kind of annoying me and I'm unsure how to fix it. I'm pretty sure when I was first following along with Patrick's videos, my VSCode looked exactly like his, but today when I opened it up to continue following from the 2nd video (Solidity course 7-11), my solidity colour formatting is different to his. It's bugging me. 

I'm sure it's really easy to fix but I'm a bit of a noob and need some guidance.

Thanks for your help.

![formatting](https://github.com/Cyfrin/foundry-full-course-f23/assets/145686009/80891348-b139-4550-8989-e92d48043738)

**Comment by theirrationalone** [comment]
You just need to change your vscode's theme to as patrick has.
